release.sh

```
@@ -0,0 +1,46 @@
+CURRENT_VERSION=$(ruby -r ./lib/swift_ui_rails/version.rb -e "puts SwiftUIRails::VERSION")
+echo "Current version: $CURRENT_VERSION"
+
+# Check for uncommitted changes
+if ! git diff-index --quiet HEAD --; then
+    echo "❌ Error: You have uncommitted changes. Please commit or stash them first."
+    exit 1
+fi
+
+# Run tests
+echo "🧪 Running tests..."
+cd test_app
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)